### PR TITLE
updated naming in README and addding the arxiv manuscript

### PR DESCRIPTION
**Pull request by ondrejkrejci**
Fixes: #266 
On top of that the arxiv manuscript is mentioned.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +80,7 @@ Since 2014 PPM developed into the toolbox of various methodologies adjusted for
 
 * **master_backup** - Old `master` branch was recently significantly updated and named `main`. For users who miss the old master branch, we provided a backup copy. However, this version is very old and its use is discouraged. If you miss some functionality or are not satisfied with the behavior of current `main` branch please let us know by creating an *issue*.
 * **PhotonMap** - implements the latest developments concerning sub-molecular scanning probe combined with Raman spectroscopy (TERS)y and fluorescent spectroscopy (LSTM).
-* **complex_tip** - Modification of probe-particle model with 2 particles allows a better fit to experimental results at the cost of additional fitting parameters.
+* **complex_tip** - Modification of the Probe-Particle <odel with 2 particles allows a better fit to experimental results at the cost of additional fitting parameters.
```
Comment (NikoOinonen) [inline README.md:83]:
```suggestion
* **complex_tip** - Modification of the Probe-Particle model with 2 particles allows a better fit to experimental results at the cost of additional fitting parameters.
```

**Review by NikoOinonen — CHANGES_REQUESTED** [review]
One typo, otherwise looks good.

**Review by ProkopHapala — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -5,9 +5,11 @@
 [![PyPI-downloads](https://img.shields.io/pypi/dm/ppafm.svg?style=flat)](https://pypistats.org/packages/ppafm)
 
 
-# Probe Particle Model (PPM)
+# Probe-Particle Model
 
-Simple and efficient **simulation software for high-resolution atomic force microscopy** (**HR-AFM**) and other scanning probe microscopy (SPM) techniques with sub-molecular resolution (STM, IETS, TERS). It simulates deflection of the particle attached to the tip (typically CO molecule, but also e.g. Xe, Cl-, H2O and others).
+Simple and efficient **simulation software for high-resolution atomic force microscopy** (**HR-AFM**) and other scanning probe microscopy (SPM) techniques with sub-molecular resolution (STM, IETS, TERS).
+It simulates deflection of the particle attached to the tip (typically CO molecule, but also e.g. Xe, Cl-, H2O and others).
+The python package is named as `ppafm`.
```
Comment (yakutovicha) [inline README.md:12]:
```suggestion
The Python package is named as `ppafm`.
```

File: README.md [hunk 3]
```diff
@@ -5,9 +5,11 @@
 [![PyPI-downloads](https://img.shields.io/pypi/dm/ppafm.svg?style=flat)](https://pypistats.org/packages/ppafm)
 
 
-# Probe Particle Model (PPM)
+# Probe-Particle Model
 
-Simple and efficient **simulation software for high-resolution atomic force microscopy** (**HR-AFM**) and other scanning probe microscopy (SPM) techniques with sub-molecular resolution (STM, IETS, TERS). It simulates deflection of the particle attached to the tip (typically CO molecule, but also e.g. Xe, Cl-, H2O and others).
+Simple and efficient **simulation software for high-resolution atomic force microscopy** (**HR-AFM**) and other scanning probe microscopy (SPM) techniques with sub-molecular resolution (STM, IETS, TERS).
```
Comment (yakutovicha) [inline README.md:10]:
```suggestion
Simple and efficient simulation software for high-resolution atomic force microscopy (HR-AFM) and other scanning probe microscopy (SPM) techniques with sub-molecular resolution (STM, IETS, TERS).
```

I feel there is too much bold text in the readme. Feels a bit messy to me.

File: README.md [hunk 4]
```diff
@@ -5,9 +5,11 @@
...
+It simulates deflection of the particle attached to the tip (typically CO molecule, but also e.g. Xe, Cl-, H2O and others).
```
Comment (yakutovicha) [inline README.md:11]:
```suggestion
It simulates the deflection of the probe particle attached to the tip (typically CO molecule, but also e.g. Xe, Cl-, H2O and others).
```

maybe we should mention the "probe particle" here?

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks, @ondrejkrejci. Here is my 50 cents

**Comment by yakutovicha** [comment]
@ondrejkrejci would you be able to address those issues before tomorrow's meeting? Thanks!

**Comment by ondrejkrejci** [comment]
Don't know, let's see

**Comment by ondrejkrejci** [comment]
Ok, I have managed :-)

**Review by NikoOinonen — APPROVED** [review]

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**